code/modules/mob/living/simple_animal/slime/death.dm

```diff
@@ -4,11 +4,18 @@
 	if(!gibbed)
 		if(is_adult)
 			var/mob/living/simple_animal/slime/M = new(loc, colour)
+			M.happiness = 0
+			if(!ckey && happiness < 0 && prob(abs(happiness)*2))//so at -20 its a 40% chance. at -50 happiness it's an 100% chance
+				M.death()
+				M.desc += " It looks like it was a stillborn..."
```

Comment:
this won't show up due to how examining mobs works, you'll need to make it work with the examine proc

Reply:
> I left some of the slimes for a few minutes... now all of them are dead! o-o

Slimes don't die from being unhappy. Did you put them in a cold room?

Reply:
Ah fuck i put that in the wrong place...

@SomeguyManperson it does show up tho >.>

Reply:
No! They're all dead in their pen! [https://gyazo.com/a11a94a7319301c046e0607455798dce.png](url)